src/app/Services/ResourceServices.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const ResourceServices = () => {
+  return (
+    <div className="relative h-screen w-full">
+      <div className="h-971px w-1116px absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 transform rounded-[40px] border-[4px] border-[#93B48F] bg-[#FFFBF9] text-center font-[Besley] text-[24px]">
```

Comment:
use a tailwind preset "rounded", don't hard code

<img width="104" alt="image" src="https://github.com/user-attachments/assets/27bde376-aee0-484f-a411-8d5088240623" />

Reply:
And for the font issue, I have someone else importing the exact font so when thats approved I'll have you fix that later here.

Reply:
This is how it currently looks:
<img width="1443" alt="image" src="https://github.com/user-attachments/assets/099036a7-43ed-4a8a-9530-98b785bd29d7" />

But it has to look like this:
<img width="833" alt="image" src="https://github.com/user-attachments/assets/c67b0db1-2f48-4b06-874a-2c106f5dbabb" />
And fit almost the ENTIRE width of the screen